src/python/grpcio_tests/tests/interop/server_ssl_cert_reload_test/README.md

```diff
@@ -0,0 +1,24 @@
+This tests the ability of gRPC server to rotate its SSL cert for use
```

Comment:
Rather than a README, just put this prose into the module doc string of the test?

Reply:
sure, moving to `_test.py`

Reply:
done